docs/pages/serving_an_origin.mdx

```diff
@@ -55,123 +57,309 @@ Once you go through the process, locate your credential files and set the follow
 
-## Serve an Origin
+## Serving Origins
+
+When you've completed the aforementioned steps, you're ready to start configuring the origin that will add your data to a federation. Serving an origin is the process of taking some underlying storage repository and making its data accessible via a namespace prefix in your federation. For example, you might make files in the directory `/my/directory` available at the federation path `/my/namespace` so that anyone with access to the federation can get objects from the directory
+
+By default, Pelican origins serve files from a POSIX backend, the filesystem used by Linux computers. However, Pelican aims to support a variety of backends and we currently also support serving objects from S3. Configuration for S3 is mostly similar to configuration for POSIX filesystems, but with a few important differences. For information about S3 backends, jump to our [s3 documentation](#launch-the-origin-with-an-s3-storage-backend)
 
 > If you are running Pelican docker image to serve an origin, please refer to the [docker image documentation](./install/docker.mdx#run-pelican-origin-server).
 
 ### Find a federation to join
 
-Before serving an origin, you need to find a Pelican federation to join in. If you are unfamiliar with the term **federation**, refer to [Useful Terminology](./client-usage.mdx#useful-terminology) before proceeding.
+Before serving an origin, you need to decide which [**federation**](./client-usage.mdx#useful-terminology) your data will be accessed through. For example, the Open Science Data Federation (OSDF) is Pelican's flagship federation, and if you are interesting in serving an OSDF origin, you can refer to the [OSDF website](https://osg-htc.org/) for details about how to join.
+
+Federations are identified by a URL, which is used to host information your origin will need in order to discover other federation services. For example, the OSDF's federation URL is `https://osg-htc.org`, and an origin that joins the OSDF will visit `https://osg-htc.org/.well-known/pelican-configuration` to get important metadata about the federation's central services (the Director and Registry).
 
-If you don't have a federation in mind, the Open Science Data Federation (OSDF) is an example Pelican federation that you can join in for testing purposes. If you are interesting in serving an OSDF origin, refer to the [OSDF website](https://osg-htc.org/) for details.
+To point your origin at a specific federation, you can either pass the `-f <federation URL>` flag if running from the command line, or configure `Federation.DiscoveryUrl: <federation URL>` in your config yaml.
 
-The federation discovery URL for OSDF is `osg-htc.org`. You may use this as your `<federation>` argument in the next section when launching your origin.
+### Serve the Origin
```

Comment:
I avoid using "serve" here to distinguish between the secondary title for this section "Serve Origins". Up to you to decide which one to use